src/formatDate.js

```diff
@@ -48,9 +48,38 @@
  *
  * @returns {string}
  */
-
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const oldSeparator = fromFormat[3];
+  const newSeparator = toFormat[3];
+
+  const oldDateArr = date.split(oldSeparator);
+  const newDateArr = new Array(3);
+
+  for (let i = 0; i < newDateArr.length; i++) {
+    let oldDateItem = oldDateArr[i];
+    const oldFormatItem = fromFormat[i];
+    let newFormatIndex = toFormat.indexOf(oldFormatItem);
+
+    if (newFormatIndex < 0) {
+      newFormatIndex = oldFormatItem === 'YY' ? toFormat.indexOf('YYYY')
+        : toFormat.indexOf('YY');
```

Comment:
if you move one part of the ternary operator to the new line then move another part also
```suggestion
      newFormatIndex = oldFormatItem === 'YY'
        ? toFormat.indexOf('YYYY')
        : toFormat.indexOf('YY');
```

Reply:
Thanks, will change this tomorrow!